NATS.Jwt/NatsJwt.cs

```diff
@@ -378,6 +378,38 @@ public NatsActivationClaims DecodeActivationClaims(string jwt)
         return claims;
     }
 
+    /// <summary>
+    /// Sets the user claims as scoped or not.
+    /// </summary>
+    /// <param name="userClaims">The user claims to modify.</param>
+    /// <param name="scoped">Indicates whether the user claims should be scoped or not.</param>
+    public void SetScopedUserClaims(NatsUserClaims userClaims, bool scoped)
```

Comment:
Do you think could this be better as an extension methof?

Reply:
I thought of that. `NatsUserClaims` had no methods. `NatsJwt` had methods that operated on it, but looking at it again, it looks like it's just Encode/Decode. I'll move to an extension.